File: C# Fundamentals/C# OOP Basics/09. Inheritance/Ex/P01_Person/Child.cs
using System;
using System.Collections.Generic;
using System.Text;

public class Child : Person
{
    public Child(string name, int age) : base(name, age)
    {

    }
    public override int Age
    {
        get
        {
            return base.Age;
        }
        set
        {
            if(value > 15)
            {
                throw new InvalidOperationException("Child's age must be less than 15!");
            }
            base.Age = value;
        }
    }
    public override string ToString()
    {
        StringBuilder stringBuilder = new StringBuilder();
        stringBuilder.Append(String.Format("Name: {0}, Age: {1}",
                             this.Name,
                             this.Age));

        return stringBuilder.ToString();

    }
}

File: Java/01.Advanced/01.Advanced/11.Classes/Ex/src/t_07_google/Child.java
package t_07_google;

public class Child {
    private final String name;
    private final String birthDate;

    public Child(String name, String birthDate) {
        this.name = name;
        this.birthDate = birthDate;
    }

    @Override
    public String toString() {
        return String.format("%s %s%n", name, birthDate);
    }
}
